features/bevragen/persoon/verblijfplaats/dev/autorisatie-gba.feature

```diff
@@ -0,0 +1,300 @@
+      Dan heeft de response een persoon met de volgende 'verblijfplaats' gegevens
+      | naam                     | waarde          |
+      | straat                   | Borgesiusstraat |
+      | datumAanvangAdreshouding | 19860801        |
+
+
+  Rule: de 'verblijfplaatsBinnenland' field alias moet worden gebruikt door een consumer die niet is geautoriseerd voor het bevragen van 'verblijfplaats buitenland' velden
+
+    Scenario: Afnemer vraagt om alleen binnengemeentelijke verblijfplaatsgegevens en is niet geautoriseerd voor verblijf buitenland
+      Gegeven de afnemer met indicatie '12345' heeft de volgende 'autorisatie' gegevens
+      | Rubrieknummer ad hoc (35.95.60)                                   | Medium ad hoc (35.95.67) | Datum ingang (35.99.98) |
+      | 10120 80910 81110 81115 81120 81130 81140 81150 81160 81170 81210 | N                        | 20201128                |
+      En de geauthenticeerde consumer heeft de volgende 'claim' gegevens
+      | naam         | waarde |
+      | afnemerID    | 12345  |
+      | gemeenteCode | 0518   |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                                                     |
+      | type                | RaadpleegMetBurgerservicenummer                            |
+      | burgerservicenummer | 000000024                                                  |
+      | fields              | burgerservicenummer,verblijfplaatsBinnenland.verblijfadres |
+      Dan heeft de response een persoon met de volgende 'verblijfplaats' gegevens
+      | naam                 | waarde          |
+      | straat               | Borgesiusstraat |
+      | huisnummer           | 103             |
+      | huisletter           | b               |
+      | huisnummertoevoeging | 2               |
+      | postcode             | 2497BV          |
+      | woonplaats           | Scheveningen    |
+
+    Scenario: Afnemer vraagt om alleen binnengemeentelijke datumVan en is niet geautoriseerd voor verblijf buitenland
+      Gegeven de afnemer met indicatie '12345' heeft de volgende 'autorisatie' gegevens
+      | Rubrieknummer ad hoc (35.95.60) | Medium ad hoc (35.95.67) | Datum ingang (35.99.98) |
+      | 10120 81010 81030 81110 81210   | N                        | 20201128                |
+      En de geauthenticeerde consumer heeft de volgende 'claim' gegevens
+      | naam         | waarde |
+      | afnemerID    | 12345  |
+      | gemeenteCode | 0518   |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                                                |
+      | type                | RaadpleegMetBurgerservicenummer                       |
+      | burgerservicenummer | 000000024                                             |
+      | fields              | burgerservicenummer,verblijfplaatsBinnenland.datumVan |
+      Dan heeft de response een object met de volgende gegevens
```

Comment:
Moet hier niet ook gedefinieerd worden dat de response een burgerservicenummer bevat? Deze opmerking geldt ook voor andere plaatsen in dit en andere feature bestanden.
Deze dan regel is sowieso niet correct.

Reply:
burgerservicenummer uit fields gehaald

Dan stap is gecorrigeerd. Had ik blijkbaar uit een oud scenario gekopieerd.